modules/spring-config/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what is this?

Reply:
This file enables a feature to allow mocking final methods (more info [here](https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods)).  We need it to mock `beanFactory.getSingletonMutex()`